nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh

```diff
@@ -35,6 +35,10 @@ else
   JAVA_PROMETHEUS_OPTS="-javaagent:$agentPath=$PROMETHEUS_METRICS_PORT:$PROMETHEUS_AGENT_CONFIG_FILE"
 fi
 
+if [ "$USAGE_REPORTS_SOURCE" == "" ]; then
```

Comment:
I wonder if we need to allow overrinding it? Or it's just in case?

Reply:
It is intensional. "docker" is too much general. I want to override it by "quickstart-docker-compose", "helmchart"

Reply:
:+1: